quickstart/201-azfw-multi-addresses/outputs.tf

```diff
@@ -0,0 +1,8 @@
+output "resource_group_name" {
+  value = azurerm_resource_group.rg.name
+}
+output "backend_admin_password" {
+  sensitive = true
+  value     = azurerm_windows_virtual_machine.vm_backend.*.admin_password
```

Comment:
Please use `azurerm_windows_virtual_machine.vm_backend[*].admin_password` instead.

Reply:
I believe I tried that and kept getting an error

Reply:
Hi @cshea15 I just tried this output and it worked. 
<img width="493" alt="image" src="https://github.com/Azure/terraform/assets/2233414/fce2dc6c-d61b-4d0a-8c37-66dfd9039c87">

Reply:
was able to deploy with your recommendation, not sure what I was doing wrong before. Thanks!